spec/javascripts/components/option-select-spec.js

```diff
@@ -236,45 +243,41 @@ describe('An option select component', function () {
   })
 
   describe('setupHeight', function () {
-    var $checkboxList, $checkboxListInner
+    var $checkboxList, $checkboxListInner, $checkboxLabel
 
     beforeEach(function () {
       $element = $(html)
       $('body').append($element)
 
       // Set some visual properties which are done in the CSS IRL
-      $checkboxList = $element.find('.js-options-container')
-      $checkboxList.css({
-        height: 200,
-        position: 'relative',
-        overflow: 'scroll'
-      })
-      $checkboxList.find('label').css({
-        display: 'block'
-      })
-
-      $checkboxListInner = $checkboxList.find(' > .js-auto-height-inner')
+      $checkboxList = $element.find('.js-options-container')[0]
+      $checkboxList.style.height = 200
+      $checkboxList.style.position = 'relative'
+      $checkboxList.style.overflow = 'scroll'
+
+      $checkboxLabel = $checkboxList.querySelector('label')
+      $checkboxLabel.style.display = 'block'
+
+      $checkboxListInner = $checkboxList.querySelector('.js-auto-height-inner')
```

Comment:
Yes, this is not an ideal situation!  I have reverted where possible.  However, some JQuery/Vanilla JS areas still exist.  I suspect this may be inevitable until we also convert the tests to Vanilla JS.

Reply:
Generally we don't need to remove jQuery from tests because Jasmine includes its own jQuery that doesn't get included in the public frontend code, and writing tests with jQuery is often easier than with vanilla JS. However tests will need to be updated where (for example) jQuery objects are passed to non jQuery code, or jQuery events are triggered on non jQuery objects.

I agree that it's not ideal to have a mix of both in the tests, but while the effort to remove jQuery from our production code is huge, I suspect the effort to remove it from the tests would be bigger. I'd rather keep the tests as unchanged as possible now, to give us a good guidepost that removing jQuery hasn't broken anything, then look at updating the tests later.